functions/snippets/src/main/java/SendHttpRequest.java

```diff
@@ -15,36 +15,39 @@
  */
 
 // [START functions_concepts_requests]
+
+import com.google.cloud.functions.HttpFunction;
+import com.google.cloud.functions.HttpRequest;
+import com.google.cloud.functions.HttpResponse;
+
+import java.io.BufferedWriter;
 import java.io.IOException;
-import java.io.PrintWriter;
 import java.net.URI;
 import java.net.http.HttpClient;
-import java.net.http.HttpRequest;
-import java.net.http.HttpResponse;
 import java.net.http.HttpResponse.BodyHandlers;
 import java.time.Duration;
-import javax.servlet.http.HttpServletRequest;
-import javax.servlet.http.HttpServletResponse;
 
-public class SendHttpRequest {
+public class SendHttpRequest implements HttpFunction {
 
   // Create a client with some reasonable defaults. This client can be reused for multiple requests.
   private static HttpClient client =
       HttpClient.newBuilder().connectTimeout(Duration.ofSeconds(10)).build();
 
-  public void sendHttpRequest(HttpServletRequest request, HttpServletResponse response)
+  @Override
+  public void service(HttpRequest request, HttpResponse response)
       throws IOException, InterruptedException {
     // Create a GET sendHttpRequest to "http://example.com"
     String url = "http://example.com";
-    HttpRequest getRequest = HttpRequest.newBuilder().uri(URI.create(url)).GET().build();
+    java.net.http.HttpRequest getRequest =
+        java.net.http.HttpRequest.newBuilder().uri(URI.create(url)).GET().build();
 
     // Send the sendHttpRequest using the client
-    HttpResponse<String> getResponse = client.send(getRequest, BodyHandlers.ofString());
+    java.net.http.HttpResponse<String> getResponse =
```

Comment:
Please `import java.net.http;` and use `http.HttpResponse`.

Reply:
That's not possible. Java doesn't allow you to import packages. You can either import `com.google.cloud.functions.HttpResponse` or `java.net.http.HttpResponse`, but not both. The name conflict is a bit unfortunate, but I'm not sure it's likely to be common enough to prompt us to change the name of the class in the FF API.